.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    bracketSpacing: false,
+    jsxBracketSameLine: true,
+    singleQuote: true,
+    trailingComma: 'all',
+};
```

Comment:
this is not the same as the prettier config from our code-standards package

Reply:
I'm not seeing a `.prettierrc.js` file in the code-standards repo. Am I not supposed to have this at all?

Reply:
Prettier should be configured according to the usage instructions in the prettier-config package (https://github.com/pxblue/code-standards/tree/dev/prettier-config#configuration). So there should not be a prettierrc file...the configuration should be coming from the node_module. This should have been hooked up automatically by the CLI.